monkey-boo-boo/Trestlebridge-Farms
# can view animal count by type

# Description

When a user attempts to purchase an animal and place in a grazing field, they can see the number of total animals in that field as well as the count of each animal type.

## Type of change

Check all that apply.

- [ ] This pull request fixes a bug 
- [x] This pull request adds a new feature
- [ ] This pull request will break existing functionality

# Testing Instructions

Try to create a grazing field and then purchase multiple animals - see if the field displays animals by type/count.

# Checklist:
